gslib/commands/du.py

```diff
@@ -113,6 +115,21 @@
       gsutil -o GSUtil:default_project_id=project-name du -shc
 """)
 
+LOCAL_ENC = locale.getpreferredencoding(False)
```

Comment:
If we only use this RunCommand, can we just define it as local variable there?

Reply:
Both this and `_GetPatternExclusion` can be removed. Changes in #674 .